### DISCUSSION TITLE
Restart IGMP Proxy

### DISCUSSION THREAD

**Discussion by xHuubx** [fabianishere/udm-iptv#192] [General]
Hello Fabian, is there a command to restart the IGMP Proxy?
Sometimes I get a error like "_igmpproxy: MC-Router API already in use; Errno(98): Address already in use"
Restarting the UDMP doesn't help, is there a command to release/restart that process?

**Comment by fabianishere** [comment]
Did you convert from the old Podman-based setup by any chance?

In that case, you need to remove the boot script in `/mnt/data/on_boot.d` as well.

**Comment by xHuubx** [comment]
Okay, I had to do a factory reset, installed your tool all over again. I guess that means a clean install right?
So there is no restart option/command?

**Reply by fabianishere** [reply]
You can restart the service as follows:

```bash
systemctl restart udm-iptv
```

However, the error you are getting indicates that another instance of `igmpproxy` is running. Either you still have the Podman-based setup running on your device or you have enabled the `igmpproxy` proxy, which you can disable as follows:

```bash
systemctl disable --now igmpproxy
 ```

**Reply by xHuubx** [reply]
Great! I will try that next time. I really appreciate your help.

Last question what would be the command to remove the old Podman boot script?

**Reply by fabianishere** [reply]
I guess `rm /mnt/data/on_boot.d/15-iptv.sh`.

**Reply by xHuubx** [reply]
👍🏻
